packages/website/src/OneDemoToRuleThemAll.tsx

```diff
@@ -24,9 +25,17 @@ const Header = () => (
             <img src={logo} className="header-logo" />
         </a>
         <div className="flex space-around search">
-            <EngineProvider>
-                <StandaloneSearchBar id="header" />
-            </EngineProvider>
+            {/*
+                To toggle the feature flag, copy and paste those commands in the dev tool console:
+                sessionStorage.setItem('ff_plasma-search-bar', true) to show the bar
+                sessionStorage.setItem('ff_plasma-search-bar', false) to hide the bar
+                You need to reload the page for it to take effect.
+             */}
+            {FeatureFlags.get('plasma-search-bar') && (
+                <EngineProvider>
+                    <StandaloneSearchBar id="header" />
+                </EngineProvider>
+            )}

```

Comment:
Can we move this inside a component so that it remains transparent to this file?

Reply:
I'll try :eyes: